lib/index.erb

```diff
@@ -35,6 +36,14 @@
       <%- end -%>
       <meta name="description" content="<%= @site_map['Documents'][0].values[0][0]["SEO Description"] %>">
       <meta name="keywords" content="<%= @site_map['Documents'][0].values[0][0]["SEO Keywords"] %>">
+
+      <%- if !@site_map['Languages'].nil? && !@site_map['Documents'][0].values[0][0]["SEO Languages"].nil? -%>
+        <link rel="alternate" hreflang="<%= ISO_639.find_by_english_name(@site_map['Language']).alpha2 %>" href="<%=@site_map['Languages'][@site_map['Language']]%>" title="lang" />
+
+        <%- @site_map['Documents'][0].values[0][0]["SEO Languages"].each do |lang| -%>
+          <link rel="alternate" hreflang="<%= ISO_639.find_by_english_name(lang).alpha2 %>" href="<%=@site_map['Languages'][lang]%>" title="lang" />
```

Comment:
Similar to what was done for the other pages on line 24 of this file, shouldn't we point to the translated page by adding the slug to the end of the `href` attribute here?